src/core/bitop.d

```diff
@@ -72,20 +72,18 @@ unittest
  */
 int bsf(size_t v) pure;
 
+version(CoreDdoc)
+{
 /// ditto
+int bsf(ulong v) pure;
+}
+else version(D_LP64) { } else
```

Comment:
Well, I still think this should be `static if(size_t.sizeof == 4)` then, because your `version` constraint actually has a different meaning than the one it replaces.

But I will not argue the point further if you insist. (That'll teach me not to make mistakes in future PRs though, I guess...)